README.rst

```diff
@@ -61,6 +62,9 @@ Usage
       -t [TAGS ...], --tags [TAGS ...]
                             Regex(es) for matching tags to add a relationship link for. All tags
                             get matched by default.
+      --tags-for-inclusion [TAGS_FOR_INCLUSION ...]
+                            Regex(es) for matching tags. A test case is included if every regex
+                            matches at least one of its tags.
```

Comment:
Whats he difference between this and `tags` -> I think we need a bit more details here about this, because this also adds tag.

Reply:
`tags` determines which items, present in the `[Tags]` section, are used as traceable targets.

`tags-for-inclusion`  determines which test cases are included in the reST output.

Reply:
`tags-in-rst` or `tags-displayed-output` ?

Or maybe we beef up the help details in this way....

Reply:
Perhaps we should support filtering like Robot Framework does it: http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#tag-patterns

Thoughts?